src/sparse/cs_matrix_cholesky.rs

```diff
@@ -293,7 +293,7 @@ where
     ) -> (CsMatrix<T, D, D>, CsMatrix<T, D, D>) {
         let etree = Self::elimination_tree(m);
         let (nrows, ncols) = m.data.shape();
-        let mut rows = Vec::with_capacity(m.len());
+        let rows = Vec::with_capacity(m.len());
```

Comment:
This change makes the CI fail. It should be mutable because `&mut rows` is used bellow at line 305.